deployment/network-operator/values.yaml

```diff
@@ -179,7 +179,8 @@ ofedDriver:
       enable: true
       force: false
       podSelector: ""
-      timeoutSeconds: 0
+      # It's recommended to set a timeout to avoid infinite drain in case non-fatal error keeps happening on retries
+      timeoutSeconds: 300
```

Comment:
Do we also want to set a default timeout in the CRD itself?

Reply:
Good idea, thanks

Reply:
Let's also check with @adrianchiris and @e0ne. WDYT? Do we need to set such defaults in both: charts and CRD itself?

Reply:
discussed offline: need to apply to both

Reply:
+1 on setting both